node/src/BaseClient.ts

```diff
@@ -985,7 +986,8 @@ export class BaseClient {
             | command_request.Command
```

Comment:
Do you think it will be possible to break this function?
To make the parameters a type, so its stop growing, and to create separate functions for each request and a function return the fitting function base on type?
Or other method you think of.
I'm afraid it's start to grow and the current impl doesn't fit anymore.

Reply:
I think it's possible to break up the function as you suggested. Maybe we should proceed by handling it in a separate issue. What do you think?

Reply:
Works for me! There's a lot of code already.
Want to open an issue, or should I take it from here?

Reply:
This PR will be merged in 2 days, right? I don't think I have enough time to handle this in 2 days, so I'd like to handle it separately.

Reply:
Yes i agree, make sense.

Reply:
I've created an issue for this :)